tools/AzureRM.BootStrapper/help/Install-AzureRmProfile.md

```diff
@@ -44,6 +44,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Scope
```

Comment:
Added confirmation message to warn user of potential conflict if cmdlets were detected installed in other scope. Also added -Force param to override the message